.travis.yml

```diff
@@ -2,19 +2,15 @@ language: python
 python: 2.7
 sudo: false
 cache:
-  pip: true
   directories:
-    - $HOME/buildout-cache
+  - test.plone_addon/eggs
```

Comment:
@jensens this seems not to be working in the context of the add-on test; we may probably want to add again the `default.cfg` file with an updated configuration or just get rid of the virtualenv inside the add-on test.

it's late now and I need to wake up "early"; check if you can finish this.